lib/wizard_timeline_manager.rb

```diff
@@ -131,9 +131,11 @@ def save_block_and_gradeable(block)
     'books_handout' => ['Books', 'https://wikiedu.org/books'],
     'chemistry_handout' => ['Chemistry', 'https://wikiedu.org/chemistry'],
     'ecology_handout' => ['Ecology', 'https://wikiedu.org/ecology'],
-    'environmental_sciences_handout' => ['Environmental Sciences', 'https://wikiedu.org/environmental_sciences'],
+    'environmental_sciences_handout' => ['Environmental Sciences',
```

Comment:
This makes the layout inconsistent, and the lines aren't *that* long. I suggest wrapping this entire constant in line-length disable comments, instead of changing the layout.